tests/sars_cov_2/test_generate_pipeline_results_files_sars_cov_2.py

```diff
@@ -79,9 +81,9 @@ def test_generate_pipeline_results_files(
         "--analysis-run-name",
         analysis_run_name,
         "--metadata-file",
-        test_data_path / "pipeline_results_files" / metadata,
+        test_data_path / "pipeline_results_files" / pathogen / metadata,
```

Comment:
Same comment here about a fixture for a path that looks identical bar the final leaf.

Reply:
See commit: `PSG-2361: use fixtures for common test data paths`